server/scheduler/scheduler.go

```diff
@@ -14,9 +14,9 @@ import (
 	"github.com/bleenco/abstruse/server/core"
 	"github.com/bleenco/abstruse/server/ws"
 	"github.com/drone/go-scm/scm"
+	"github.com/golang/protobuf/proto"
 	"github.com/logrusorgru/aurora"
 	"go.uber.org/zap"
-	"google.golang.org/protobuf/encoding/protojson"
```

Comment:
Is this deprecated? I see this one is used in the services

Reply:
no, actually `github.com/golang/protobuf` package is deprecated. I just updated everything to this package including `install_dependencies` script in `Makefile`. It should work now.